META.d/_summary.yml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+schema: 1.1
+partition: tf-ecosystem
+category: library
+
+summary:
+  owner: team-tf-editor-experience
```

Comment:
From a compliance perspective, the Core team are currently considered the owners of this library. I don't mind having this be y'all, but then we should probably be consistent and have the compliance owners changed to you all as well.